docs/absolute.md

```diff
@@ -316,7 +316,8 @@ If you do need to calculate the difference between two `Temporal.Absolute`s in y
 For example, you might decide to base the calculation on your user's current time zone, or on UTC.
 
 Take care when using milliseconds, microseconds, or nanoseconds as the largest unit.
-For some dates, the resulting value may overflow `Number.MAX_SAFE_INTEGER`.
+For some durations, the resulting value may overflow `Number.MAX_SAFE_INTEGER` and will throw a `RangeError`.
```

Comment:
It won't actually throw a RangeError; it behaves like a regular JS Number. Above this value it will just be imprecise. (It's been debated whether Duration fields should be BigInts, but it was also mentioned as a positive that you could continue to calculate with it even if it was imprecise. I'd say resolving this is out of scope for the PR.)

Reply:
Ahh, got it. I'll update and clarify.

Reply:
Updated